### PR TITLE
#883 patch,Roles displayed

### PR DESCRIPTION
**Pull request by shivasheeshyadav**
#883 Issue fixed.

### CODE REVIEW

**Comment by Sumukh** [comment]
You should not change the actual roles themselves but rather how they are displayed in the form. The only file you should be changing is forms.py - https://github.com/Cal-CS-61A-Staff/ok/blob/master/server/forms.py#L161

The documentation on the form library is here: http://wtforms.simplecodes.com/docs/1.0.1/crash_course.html

**Comment by shivasheeshyadav** [comment]
@Sumukh So you mean to say that i should not change the text of the already given choices, instead add more choices ?

**Comment by Sumukh** [comment]
Not quite. The strings in the constants.py file should not be changed. If you look at line 170 of forms.py (https://github.com/Cal-CS-61A-Staff/ok/blob/master/server/forms.py#L170) you can see how the fields are rendered. Current it's` [r, r.title() for r in roles ]`  which results in `[('student', 'Student'), ('grader', 'Grader')...]` That second value is what's rendered in the form. 

We should keep the raw values of the roles (`['student', 'grader', ...]`) the same. 

Here's one approach that could work. Define a dictionary of role_names in constants.py that maps each of the role to the desired name
```
role_names = { 
   'grader': 'Tutor', 
   'student': 'Student', 
    ...
}
```
and then in the form you can do `[(r, role_names[r]) for r in VALID_ROLES)`

File: server/constants.py [hunk 1]
```diff
@@ -5,6 +5,13 @@
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
 LAB_ASSISTANT_ROLE = 'lab assistant'
+ROLES_NEW={
```
Comment (knrafto) [inline server/constants.py]: `ROLES_NEW` is not a very descriptive name. `ROLE_DISPLAY_NAMES` would be better

File: server/constants.py [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+'student':'Student',
+'grader':'Reader',
+'staff':'Teaching Assistant',
```
Comment (knrafto) [inline server/constants.py]: We should reuse `STAFF_ROLE` from above instead of repeating the string literal `'staff'`

File: server/forms.py [hunk 3]
```diff
@@ -168,7 +168,7 @@ class EnrollmentForm(BaseForm):
     section = StringField('Section',
                           validators=[validators.optional()])
     role = SelectField('Role', default=STUDENT_ROLE,
-                       choices=[(r, r.title()) for r in VALID_ROLES])
+                       choices=[(r, ROLES_NEW[r]) for r in VALID_ROLES])
```
Comment (knrafto) [inline server/forms.py]: I think simply `choices=ROLES_NEW.items()` would work, but I'm not 100% sure

**Comment by shivasheeshyadav** [comment]
@knrafto I've made all the changes as requested.

**Pull request closed without merging**